engine/flink/process/src/main/scala/pl/touk/nussknacker/engine/process/util/UserClassLoader.scala

```diff
@@ -1,15 +0,0 @@
-package pl.touk.nussknacker.engine.process.util
-
-object UserClassLoader {
```

Comment:
Shouldn't it be moved to nussknacker-flink-compatibility ?

Reply:
I think currently it was not really used, only in FlinkProcessRegistrar, where classloader was obvious...